docs/.vuepress/theme/components/Page.vue

```diff
@@ -51,6 +51,58 @@ export default {
     },
     isScriptLoaderActivated(exampleId) {
       return this.activatedExamples.includes(exampleId);
+    },
+    submitCodesandbox(submitEvent) {
+      // form data values
+      const html = submitEvent.target.elements.html.value;
+      const js = submitEvent.target.elements.js.value;
+      const css = submitEvent.target.elements.css.value;
+
+      const hyperformula = js.includes('hyperformula') ? { hyperformula: 'latest' } : {};
+
+      const cssObj = css
+        ? {
+          'style.css': {
+            content: css,
+          }
+        }
+        : {};
+
+      // generate codesandbox demo id
+      fetch('https://codesandbox.io/api/v1/sandboxes/define?json=1', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+          Accept: 'application/json',
+        },
+        body: JSON.stringify({
+          files: {
+            'package.json': {
+              content: {
+                name: 'handsontable-demo',
+                dependencies: {
+                  handsontable: 'latest',
```

Comment:
@adrianspdev I think this part might be problematic when the documentation gets archived under a version number (when a newer version gets released).
It would be a better idea to use an actual Handsontable version instead of `latest` here.

Reply:
done